### DISCUSSION TITLE
Compatibility with D&D4e?

### DISCUSSION THREAD

**Discussion by germandrewboat** [DFreds/dfreds-convenient-effects#207] [Show and tell]
Hey! First I want to say thank you for all your incredible modules--I run a 5E game and use Droppables and Convenient effects every session and I thoroughly appreciate your work.

I don't have any dev experience, but I was able to fork off of Convenient Effects to get it to launch in a DnD4e system world and I was able to make convenient effects for a whole bunch of Status effects in that system. Is there any way you can add compatibility for that system and also, if I sent you a JSON of the effects I made (also found in my forked repository I linked), could it be added as a database that loads everytime in a 4e world not under the Custom section? If you have any questions let me know, I'm also in the FoundryVTT discord as "thedrewboat [he/him]"

https://github.com/germandrewboat/dfreds-convenient-effects

**Comment by DFreds** [comment]
Very cool! I have been considering ways to support more systems. If you can suggest some way of sorting the 4e into folders, I think it would be a big help for me to get it working in other systems.

I should be able to only load the 4e or only load the 5e based on the active system, and then add other systems as I go

**Comment by germandrewboat** [comment]
[4e-custom-effects-plus-cover-concealment.zip](https://github.com/DFreds/dfreds-convenient-effects/files/10471216/4e-custom-effects-plus-cover-concealment.zip)

Awesome! Here's a zip with just the effects I made, all of them can be Conditions, aside from the 2 "Cover" and 2 "Concealment" effects I made. All icons are in Foundry's default library, so shouldn't have any issue on that front.

**Reply by DFreds** [reply]
Awesome, thanks!

**Reply by germandrewboat** [reply]
You got it! If I can do anything else, let me know! Like I said, no dev experience, but I can always test!
